build_scripts/windows_bash_nuttx.sh

```diff
@@ -86,8 +91,8 @@ else
     wget https://github.com/SolinGuo/arm-none-eabi-bash-on-win10-/raw/master/gcc-arm-none-eabi-5_4-2017q2-20170512-linux.tar.bz2
     tar -jxf gcc-arm-none-eabi-5_4-2017q2-20170512-linux.tar.bz2
     exportline="export PATH=$HOME/gcc-arm-none-eabi-5_4-2017q2/bin:\$PATH"
-    if grep -Fxq "$exportline" ~/.profile; then echo " GCC path already set." ; else echo $exportline >> ~/.profile; fi
-    . ~/.profile
+    if grep -Fxq "$exportline" ~/.bashrc; then echo " GCC path already set." ; else echo $exportline >> ~/.bashrc; fi
```

Comment:
FYI, Bash for Windows doesn't load .profile. IN current version it does (anecdotally) load .bashrc.